addons/binding/org.openhab.binding.netatmo/ESH-INF/config/config.xml

```diff
@@ -46,7 +46,41 @@
             <description>Read and Write thermostat's data.</description>
             <default>true</default>
         </parameter>
-                                                         
+               
+        <parameter name="accessWelcome" type="boolean" required="true">
+            <label>Access Welcome camera</label>
+            <description>Access welcome camera's data.</description>
+            <default>false</default>
+        </parameter>
+        
+        <parameter name="readWelcome" type="boolean" required="true">
+            <label>Read Welcome camera</label>
+            <description>Read welcome camera's data.</description>
+            <default>false</default>
+            <advanced>true</advanced>
+        </parameter>
+        
+        <parameter name="writeWelcome" type="boolean" required="true">
+            <label>Write Welcome camera</label>
+            <description>Write welcome camera's data.</description>
+            <default>false</default>
+            <advanced>true</advanced>
+        </parameter>
+        
+        <parameter name="welcomeEventThings" type="integer" required="false">
```

Comment:
The Netatmo API is my bridge, haven't implemented an own bridge, would it be better to implement a bridge for the welcome camera? Is it possible that the Netatmo api is the base bridge for the welcome bridge?